src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/VsProjectAdapterProvider.cs

```diff
@@ -67,6 +67,19 @@ public async Task<IVsProjectAdapter> CreateAdapterForFullyLoadedProjectAsync(Env
 
             var projectNames = await ProjectNames.FromDTEProjectAsync(dteProject, vsSolution);
             var fullProjectPath = dteProject.GetFullProjectPath();
+            if (fullProjectPath == null && dteProject.IsWebSite())
+            {
+                var projectDirectory = await dteProject.GetFullPathAsync();
```

Comment:
Website projects are special. They have no project file, and GetFullProjectPath returns null. We still need the project directory though.

Reply:
No freaking kidding.
It legit took me more time to figure out how/why I broke the website tests than the total time to diagnose all these particular issue and do the fix for them. 

There are so many things in our code that special case website projects.